brave/browser-laptop
# Add Payments to Bravery system menu

Close #3592

Auditors: @bradleyrichter @diracdeltas

Test Plan:

1. Disable Payments
2. Open OS menu "Bravery"
3. See menu item "Start using Brave payments..." -- click it
4. Tab opens to Prefs > Payments
5. Enable Payments
6. Close tab
7. Open OS menu "Bravery" again
8. Payments menu item now says "Brave payments"
9. Close and reopen Brave
10. Menu item still says "Brave payments"
